homeassistant/components/voip/assist_satellite.py

```diff
@@ -82,10 +83,11 @@ def async_add_device(device: VoIPDevice) -> None:
 class VoipAssistSatellite(VoIPEntity, AssistSatelliteEntity, RtpDatagramProtocol):
     """Assist satellite for VoIP devices."""
 
-    entity_description = AssistSatelliteEntityDescription(key="assist_satellite")
-    _attr_translation_key = "assist_satellite"
-    _attr_has_entity_name = True
-    _attr_name = None
```

Comment:
If it has this, it should just have the device name 🤔  where does `None` come from.

Reply:
Mine had a none in the entity id, but not the name. So maybe it got that before I added these attributes.

Reply:
On your dev env? Can you try setting up a new entity and if that has the same?

Do you have a device attached to the entity?